docs/src/logistic.md

```diff
@@ -39,10 +39,11 @@ saver = train.Saver()
 # Run training
 run(sess, global_variables_initializer())
 checkpoint_path = mktempdir()
-@info("Checkpoint files saved in $checkpoint_path")
+@info "Checkpoint files saved in $checkpoint_path"
 for epoch in 1:100
     cur_loss, _ = run(sess, (Loss, minimize_op), Dict(X=>x, Y_obs=>y))
-    println(@sprintf("Current loss is %.2f.", cur_loss))
+    cur_loss_rounded = round(cur_loss,digits=2)
+    println("Current loss is $cur_loss_rounded.")
```

Comment:
What's wrong with `@sprintf`? Did Julia 1.0 change its API or something?

Reply:
Ditto again, it would not run, so I looked and found an old discussion about sprintf and how the community was looking for some volunteer to resolve the outstanding issues. Perhaps these are now resolved. I did a `? sprintf` on my machine and found nothing of use. So I worked around it using functionality that appeared to be available. Let me know if I can help by further digging.

Reply:
It's moved to `Printf` standard library module, apparently.